pkgs/development/libraries/ffmpeg-full/default.nix

```diff
@@ -444,6 +444,12 @@ stdenv.mkDerivation rec {
     cp -a tools/qt-faststart $out/bin/
   '';
 
+  postFixup = optionalString stdenv.isLinux ''
+    # Set RUNPATH so that libnvcuvid in /run/opengl-driver(-32)/lib can be found.
+    # See the explanation in addOpenGLRunpath.
+    addOpenGLRunpath $out/lib/libavcodec.so*
```

Comment:
do you also need to add `libavutil.so*` and `libavfilter.so*`?

At least for libavutil, I also found a `libcuda.so.1^@Cannot load %s` in the binary stream…

Reply:
Thanks for checking that. The original fixup only considered libnvcuvid (only in libavcodec.so*). I've updated the PR to also handle the libcuda references (libavcodec & libavutil) for ffmpeg and ffmpeg-full. I've checked the other library files and found no further references to libnvcuvid or libcuda for ffmpeg and ffmpeg-full.